robot/Cumulus/tests/browser/automated_soft_credits/asc_reciprocal_relation.robot

```diff
@@ -2,69 +2,93 @@
+    ...                                     object_id=${data}[contact2][Id]
+    Navigate To And Validate Field Value    Total Gifts This Year                contains    $500.00    section=Soft Credit Total
+    Navigate To And Validate Field Value    Total Gifts                          contains    $500.00    section=Soft Credit Total
+    Navigate To And Validate Field Value    Soft Credit This Year                contains    $0.00      section=Soft Credit Total
+    Navigate To And Validate Field Value    Soft Credit Total                    contains    $0.00      section=Soft Credit Total
     
 Test Case 2
-    [tags]  unstable
-    &{contact1} =  API Create Contact    Email=skristem@robot.com 
-    &{contact2} =  API Create Contact    AccountId=&{contact1}[AccountId]
-    ${ns} =  Get NPSP Namespace Prefix
-    &{relation} =  API Create Relationship    &{contact1}[Id]    &{contact2}[Id]    Coworker    ${ns}Related_Opportunity_Contact_Role__c=Soft Credit
-    Go To Record Home    &{contact2}[Id]
-    Select Tab  Related
-    Click Related Table Item Link    Relationships    &{contact1}[FirstName] &{contact1}[LastName]
-    Click Button    title=Edit Related Opportunity Contact Role
-    Wait For Locator  record.footer
-    Click Flexipage Dropdown   Related Opportunity Contact Role              Soft Credit
-    Click Button    Save
-    &{opportunity} =  API Create Opportunity    &{contact1}[AccountId]    Donation    Name=Reciprocal test $500 donation    Amount=500    ${ns}Primary_Contact__c=&{contact1}[Id]  
-    Go To Record Home    &{opportunity}[Id]
+    [Documentation]            Create 2 contacts on same household and Create relationship with contact2 and Related OCR as Soft Credit via API.
+    ...                        On the reciprocal relation add Related Opp contact role as Soft Credit.Create an opp with contact1 as primary.
+    ...                        Verify contact1 is Donor, contact2 has soft credit role. Run donations batch job 
+    ...                        verify contact1 gets hard credits and no soft credits, contact2 gets soft credits
+    [tags]                     feature:Automated Soft Credits       W-039819
+    
+    Go To Page                              Details                              Contact                                
+    ...                                     object_id=${data}[contact4][Id]
     Select Tab  Related
-    Select Relatedlist    Contact Roles
+    Click Related Table Item Link           Relationships                        ${data}[contact3][FirstName] ${data}[contact3][LastName]
+    Edit Record Dropdown Value             	Related Opportunity Contact Role     Soft Credit
+    Save Record
+    &{opportunity2} =                       API Create Opportunity               ${data}[contact3][AccountId]    Donation    
+    ...                                     Name=Reciprocal test $500 donation    
```

Comment:
this can be in setup ?

Reply:
no it cant be in setup. Other stuff has to happen before this opp can be created